src/components/organisms/GlobalNav/GlobalNav.js

```diff
@@ -85,7 +90,6 @@ const GlobalNav = forwardRef((props, globalNavRef) => {
             />
           </GlobalNavLinkHolder>
         ))}
```

Comment:
Can you just add it below here as a link wrapped in a GlobalNavLinkHolder. It will always go down and can then just appear underneath gravy for now.

```
<GlobalNavLinkHolder position={navDownPosition}>
          <a href="">{manualKerning("Insta")}</a>
</GlobalNavLinkHolder>
```

Reply:
Maybe classname button?

Reply:
Done below, styling added.